11.0.Dockerfile

```diff
@@ -20,8 +20,8 @@ ENV DB_FILTER=.* \
     OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
     PATH="/home/odoo/.local/bin:$PATH" \
     PIP_NO_CACHE_DIR=0 \
-    PTVSD_ARGS="--host 0.0.0.0 --port 6899 --wait --multiprocess" \
-    PTVSD_ENABLE=0 \
+    DEBUGPY_ARGS="--listen 0.0.0.0:6899 --wait-for-client" \
```

Comment:
Thinking this a second time, we need a way to support backwards compatibility.

Otherwise, the only way to debug would be that the project is updated to the latest template and the latest image *or* based on an older image and template.

I'd prefer it if everything just works, so even if `PTVSD_` is no longer a cool prefix, I think we should still support it for a while.

For v14, after merging #323, we can support only `DEBUGPY_`, because there should exist no v14 projects out there.

Reply:
Yes, that makes sense. I will make the changes.

Reply:
Changes made. I also needed to change the copier template, as I changed the port where `debugpy` will run. See the related PR to see the changes